lib/src/modules/macho/mod.rs

```diff
@@ -446,6 +446,39 @@ fn import_hash(ctx: &mut ScanContext) -> Option<RuntimeString> {
     Some(RuntimeString::new(digest))
 }
 
+/// Returns an md5 hash of the symbol table in the mach-o binary
+#[module_export]
+fn sym_hash(ctx: &mut ScanContext) -> Option<RuntimeString> {
+    let macho = ctx.module_output::<Macho>()?;
+    let mut md5_hash = Md5::new();
```

Comment:
Allocate the `Md5` object after the initial validation. In cases were the file doesn't have a symtab the allocation is not needed.

Reply:
Fixed 😄